app/models/container_node.rb

```diff
@@ -79,4 +80,19 @@ def ipaddress
   def cockpit_url
     URI::HTTP.build(:host => ipaddress, :port => 9090)
   end
+
+  supports :common_logging do
```

Comment:
Not sure I like the phrase "common logging" as that is a Red Hat idiom. cc @blomquisg